content/code/markdown.md

```diff
@@ -2,14 +2,31 @@
 
 ## General
 
+* Always keep the scope of the document in mind. A document should precisely fulfill its purpose, nothing more, nothing less.
+  It is a common pitfall to end up going into rabbit holes and spending half a document explaining irrelevant details.
+* Try to write short sentences whenever possible to avoid complex grammar, complex use of tenses, ambiguous pronouns and so on.
+  A good guideline when it comes to technical writing is to aim for 20-30 words per sentence.
+  Keeping sentences short should however never come at the expense of clarity, syntactic cues and important information.
+* Just like in code, consistency is key.
+
+### Sections
+
 * Section titles should follow the [Chicago Title Capitalization](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style) standard.
-* Lists should use the `*` character rather than the `-` character.
 * Documents should start with a level one heading and should ideally be the same as the file name.
-* For long files that are not served within this repository, it is best to have a table of contents at the end of the introduction of the level one heading section.
+* Sections should be ordered hierarchically. Each document starts with a level one heading (`#`), which contains one or more level two headings (`##`), which contain one or more level threes (`###`) and so on.
```

Comment:
Regarding the headings, shouldn't it be that they _might_ contain one or more level N headers?

E.g. I have a doc with a level one heading, and inside it probably one or more level two headings. Those level two headings can sit by themselves, with content only, with no level three headings inside.

So to clarify, it's just the matter of _might contain_ vs _contain_ .